java/semantickernel-connectors-parent/semantickernel-connectors/src/main/java/com/microsoft/semantickernel/connectors/ai/openai/util/OpenAIClientSettings.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+package com.microsoft.semantickernel.connectors.ai.openai.util;
+
+import com.microsoft.semantickernel.exceptions.ConfigurationException;
+
+public abstract class OpenAIClientSettings {
```

Comment:
May want to add a javadoc stating that this class is primarily for `OpenAI` and `Azure OpenAI` clients.